apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/model/mapper/OrganizationMapperTest.java

```diff
@@ -0,0 +1,111 @@
+package org.sagebionetworks.openchallenges.organization.service.model.mapper;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.sagebionetworks.openchallenges.organization.service.model.dto.OrganizationDto;
+import org.sagebionetworks.openchallenges.organization.service.model.entity.OrganizationEntity;
+
+public class OrganizationMapperTest {
+  @Test
+  public void ConvertToEntity_ShouldReturnDtoProperties_WhenDtoPropertiesPassed() {
+
+    // create new Dto with properties to be copied to organization entity
+    OrganizationDto dTo = new OrganizationDto();
```

Comment:
Don't make up new spellings: here use `dto` or `organization`.

Reply:
Ok I thought I needed to capitalize the T in to since it was a variable name.

Reply:
The only difference between a variable name and a class name is the first letter (capitalized for class name).

Reply:
Updated the capitalization of `dTo` to all lowercase.